cmd/genji/shell/command.go

```diff
@@ -16,13 +16,45 @@ import (
 var commands = []struct {
 	Name        string
 	Options     string
+	DisplayName string
 	Description string
+	Aliases     []string
 }{
-	{".exit", ``, "Exit this program."},
-	{".help", ``, "List all commands."},
-	{".tables", ``, "List names of tables."},
-	{".indexes", `[table_name]`, "Display all indexes or the indexes of the given table name."},
-	{".dump", `[table_name]`, "Dump database content or table content as SQL statements"},
+	{
+		Name:        ".exit",
+		Options:     "",
+		DisplayName: ".exit or exit",
+		Description: "Exit this program.",
+		Aliases:     []string{"exit"},
+	},
+	{
+		Name:        ".help",
+		Options:     "",
+		DisplayName: ".help or help",
+		Description: "List all commands.",
+		Aliases:     []string{"help"},
+	},
+	{
+		Name:        ".tables",
+		Options:     "",
```

Comment:
nit: You don't have to do this in Go, non initialized fields are equal to the [zero value of their type](https://www.geeksforgeeks.org/zero-value-in-golang/)

```suggestion
```